### PR TITLE
HTS-394: Implement GA analytics 

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

**Comment by sureshhmrc** [comment]
@jamesbirrellgray fyi

File: app/uk/gov/hmrc/helptosavefrontend/views/helpers/user_details.scala.html [hunk 1]
```diff
@@ -35,8 +35,13 @@ <h4 class="heading-small">@Messages("hts.register.check-and-confirm-your-details
 </div>
 <div class="divider--bottom">
     <h4 class="heading-small">@Messages("hts.register.check-and-confirm-your-details.email")
-        <span><a style="float: right;
-            font-weight: normal;" href=@routes.UpdateEmailAddressController.getUpdateYourEmailAddress()>@Messages("hts.register.check-and-confirm-your-details.change-button-label")</a></span>
+        <span><a style="float: right; font-weight: normal;"
+                 href=@routes.UpdateEmailAddressController.getUpdateYourEmailAddress()
+                 data-ga-event-category = "links"
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/helptosavefrontend/views/helpers/user_details.scala.html]: no idea why it shows  blank lines above

File: app/uk/gov/hmrc/helptosavefrontend/views/introduction/apply.scala.html [hunk 2]
```diff
@@ -26,24 +26,32 @@
...
+                                                              >@Messages("hts.introduction.apply.ul-1-1-2")</a></li>
             <li>@Messages("hts.introduction.apply.ul-1-2-1") @Messages("hts.introduction.apply.ul-1-2-2") @Messages("hts.introduction.apply.ul-1-2-3")</li>
         </ul>
         <p>@Messages("hts.introduction.apply.par-2")</p>
         <p>@Messages("hts.introduction.apply.par-3")</p>
         <p>@Messages("hts.introduction.apply.par-4")</p>
         @form(routes.EligibilityCheckController.getCheckEligibility(), 'class -> "group") {
-            @submit('_buttonClass -> "button--get-started") {
+            @submit('_buttonClass -> "button--get-started"
+              //TODO: '_data-ga-event-category -> "links" 'data-ga-event-action = "Start Now" 'data-ga-event-label= "Apply page"
```
Comment (sureshhmrc) [inline app/uk/gov/hmrc/helptosavefrontend/views/introduction/apply.scala.html]: not sure how to fix this ,  adding attributes did not work for `@submit` or `@form`

**Comment by jamesbirrellgray** [comment]
LGTM

**Pull request merged**